src/foam/u2/view/ChoiceWithOtherView.js

```diff
@@ -22,6 +22,15 @@ foam.CLASS({
       name: 'otherKey',
       required: true
     },
+    {
+      documentation: `
```

Comment:
Fix after launch.

This is a weird implementation detail to expose to the user.  We should be maintaining the choices of the choice view to contain an "other" choice regardless of whether the choices come from a DAO or a choices property.